account_asset_management/models/account_move.py

```diff
@@ -93,6 +93,8 @@ def action_post(self):
                     raise UserError(
                         _("Asset name must be set in the label of the line.")
                     )
+                if aml.asset_id:
```

Comment:
Are you sure this is not provoking any other side effect? Shouldn't this be fixed at `account_asset_transfer` level instead?

Reply:
This is the surest fix of all fixes I come up with in this issue.